src/Storage/Device/Local.php

```diff
@@ -69,46 +72,112 @@ public function getPath($filename): string
      *
      * @param string $source
      * @param string $path
+     * @param int $chunk
+     * @param int $chunks
+     * @param array $metadata
      *
      * @throws \Exception
      *
-     * @return bool
+     * @return int
```

Comment:
If we explained what `int` means in `upload` method in the abstract class, shouldn't we need to do it here too?